impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -237,6 +238,18 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
         if (Strings.hasText(props.get(DEFAULT_CLIENT_RETRY_MAX_ATTEMPTS_PROPERTY_NAME))) {
             clientConfig.setRetryMaxAttempts(Integer.parseInt(props.get(DEFAULT_CLIENT_RETRY_MAX_ATTEMPTS_PROPERTY_NAME)));
         }
+
+        clientConfig.setRequestExecutorParams(
+            props
+                .entrySet().stream()
+                .filter(x -> x.getKey().startsWith("okta.client.requestExecutor"))
+                .collect(
+                    Collectors.toMap(
+                        x -> x.getKey().replaceAll("^okta\\.client\\.requestExecutor\\.", ""),
+                        x -> x.getValue()
+                    )
+                )
+        );
```

Comment:
Add tests to make sure this works with environment variables.

nit:
use a static var for the starts with key (include the trailing dot, otherwise `okta.client.requestExecutorBar=foo` would also map in)
Then just substring and avoid the regex alltogether

Reply:
To make it works with env variables I need this code https://github.com/okta/okta-sdk-java/pull/519#discussion_r561118496